docs/docs/05-configuration.md

```diff
@@ -126,6 +126,8 @@ $ snowpack dev --no-bundle
   - Set to `true` if Snowpack should erase the build folder before each build.
 - **`metaDir`** | `string` | Default: `__snowpack__`
   - By default, Snowpack outputs Snowpack-related metadata such as [HMR](#hot-module-replacement) and [ENV](#environment-variables) info to a folder called `__snowpack__`. You can rename that folder with this option (e.g.: `metaDir: 'static/snowpack'`).
+- **`optimize`** | `boolean` | Default: `true`
+  - By default, Snowpack will optimize your dependencies on `snowpack build`. Disable this by setting this config option to `optimize: false`.
```

Comment:
Added a line to the docs. This would be useful if, say, they were already using terser or a bundler.